code/modules/awaymissions/maploader/reader.dm

```diff
@@ -321,7 +325,7 @@ var/global/dmm_suite/preloader/_preloader = null
 /dmm_suite/preloader/New(list/the_attributes, path)
 	.=..()
 	if(!the_attributes.len)
-		Del()
+		qdel(src)

```

Comment:
I think maploader takes place before the GC is even set up, so I'm really not sure if this will work.